src/create-yo.js

```diff
@@ -40,7 +42,15 @@ async function create(
   generatorPackage = generatorPackage.split(':').shift();
   return npx(
     npx.parseArgs(
-      argv.concat('--package', generatorPackage, 'yo', '--', generatorName),
```

Comment:
I needed to fix this... Both `yo` and target `generator-*` need to be installed into same folder. Using globally installed `yo` with temporary installation of `generator-*` made by `npx` simply didn't do any tricks on my machine due to different installation paths.

Reply:
Interesting, didn't consider this (and honestly I don't understand why it's different--is that `yo`'s fault or `libnpx`'s??).  Can you explain how your change here fixes the issue?